### PR TITLE
Updated sample to install latest package and use the default source

### PR DESCRIPTION
**Pull request by niemyjski**
The specified source 'nuget.org' is invalid. Please provide a valid source.
Parameter name: source

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/Visual-Studio-Extensibility/NuGet-API-in-Visual-Studio.md](https://github.com/niemyjski/docs.microsoft.com-nuget/blob/patch-1/docs/Visual-Studio-Extensibility/NuGet-API-in-Visual-Studio.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C7%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201706071318443917-365%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/Visual-Studio-Extensibility/NuGet-API-in-Visual-Studio.md [hunk 1]
```diff
@@ -59,14 +59,14 @@ As of NuGet 3.3+, NuGet exports the following services all of which reside in th
...
     ```cs
     //Using the Import attribute
-    [Import(typeof(IVsPackageInstaller))]
-    public IVsPackageInstaller packageInstaller;
-    packageInstaller.InstallPackage("nuget.org", currentProject,
-        "Newtonsoft.Json", "9.0.1", false);
+    [Import(typeof(IVsPackageInstaller2))]
+    public IVsPackageInstaller2 packageInstaller;
+    packageInstaller.InstallLatestPackage(null, currentProject,
+        "Newtonsoft.Json", false, false);
```
Comment (joelverhagen) [inline docs/Visual-Studio-Extensibility/NuGet-API-in-Visual-Studio.md]: We should name the `bool` parameters so it is clear what they mean.

**Comment by joelverhagen** [comment]
LGTM

**Comment by kraigb** [comment]
Thanks for the contribution...sorry for taking so long to get it merged.

**Pull request merged**